tasks/main.yml

```diff
@@ -107,39 +91,80 @@
     manager: auto
   no_log: true
 
+- name: Verify if the mssql_version variable is provided correctly
+  assert:
+    that: >-
+      mssql_version | int == 2017
+      or mssql_version | int == 2019
+      or mssql_version | int == 2022
```

Comment:
it would be better to define something in `vars/main.yml` for `__mssql_supported_versions: [2017, 2019, 2022]` then you could do `that: mssql_version | int in __mssql_supported_versions`

Reply:
```suggestion
      that: mssql_version | int in __mssql_supported_versions
```

Reply:
Thank you, fixed this